src/components/relation-control/editor.scss

```
@@ -242,6 +532,267 @@
+		min-width: 16px;
+		height: 16px;
+		color: var(--maxi-grey);
+		box-sizing: border-box;
+	}
+
+	&__option-type-icon::before {
+		content: '';
+		display: block;
+		width: 12px;
+		height: 12px;
+		border: 1px solid currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)